features/welcome/index.js

```diff
@@ -1 +1,3 @@
+export * from './actions';
+export * from './constants';
 export * from './components';
```

Comment:
It seems I need to learn English alphabet better :) For some reason I though N comes before M ))